pkg/flags/flags.go

```diff
@@ -132,6 +132,9 @@ Requires setting the publish-service parameter to a valid Service reference.`)
 		electionID = flags.String("election-id", "ingress-controller-leader",
 			`Election id to use for Ingress status updates.`)
 
+		electionTTL = flags.Int("election-ttl", 30,
```

Comment:
```suggestion
		electionTTL = flags.Duration("election-ttl", 30 * time.Second,
```

Reply:
I'm really sorry about this, but the linter breaks when I apply this suggestion. Is it okay if it stays this way? It seems that your suggestion (30 * time.Second) [makes gofumpt  sad](https://github.com/kubernetes/ingress-nginx/actions/runs/8450252437/job/23146024396?pr=11142) :(

```
File is not `gofmt`-ed with `-s` (gofmt)
```

Reply:
no need to say sorry, we are all learning here